lib/icingadb/redisconnection.cpp

```diff
@@ -30,6 +30,15 @@ namespace asio = boost::asio;
 
 boost::regex RedisConnection::m_ErrAuth ("\\AERR AUTH ");
 
+RedisConnection::QueryArg::operator std::string_view() const
+{
+	if (auto str (std::get_if<String>(&m_Data)); str) {
+		return {str->CStr(), str->GetLength()};
```

Comment:
Instead of this indirections, why not implement the casting operator for the `String` class like we already do for `boost::beast::string_view()`? If we ever want to use `std::string_view` in other components as well, we would've to implement this casting each time.